examples/label_studio_annotation/materializers/pillow_image_materializer.py

```diff
@@ -0,0 +1,53 @@
+import os
+import tempfile
+from typing import Dict, Type
+
+from PIL import Image
+
+from zenml.artifacts import DataArtifact
+from zenml.io import fileio
+from zenml.logger import get_logger
+from zenml.materializers.base_materializer import BaseMaterializer
+from zenml.utils import io_utils
+
+logger = get_logger(__name__)
+
+
+class PillowImageMaterializer(BaseMaterializer):
```

Comment:
This should probably be turned into a separate Pillow integration. Or perhaps just come as a built-in materializer?
Also, if #812 makes it in, having a materializer for a single PIL image (as we would do in an integration) will be enough.

Reply:
Agreed that we should update and create a new Pillow integration / materializer once #812 is in. I'll create a ticket for it.